theme/src/main/assets/page.st

```diff
@@ -27,6 +27,15 @@ $!
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <meta http-equiv="x-ua-compatible" content="ie=edge">
     <meta name="description" content="$page.properties.("project.description")$">
+    $ if (page.properties.("project.name")) $
+        <meta property="og:title" content="$page.properties.("project.name")$">
```

Comment:
Not sure if we should use project name as `og:title`.
Shouldn't we use something like here instead?
https://github.com/sbt/sbt-paradox-material-theme/blob/4b214d82508e4ac2c7828d13a3e7706dd0c29805/theme/src/main/assets/page.st#L48-L54

Reply:
I think those were metadata of [open graph](https://ogp.me/#metadata)

Reply:
@Roiocam I understand that, however my point is if it makes sense to use the "project name" as ´og:title`. Maybe it would be better to use the same value like we use for the html `<title>` tag.

Reply:
Got it, makes sense. Thanks

Reply:
good point, changed